code-server/run.sh

```diff
@@ -79,6 +57,21 @@ if [ ! -f "$CODE_SERVER" ] || [ "${USE_CACHED}" != true ]; then
   printf "🥳 code-server has been installed in ${INSTALL_PREFIX}\n\n"
 fi
 
+# Get the list of installed extensions...
+LIST_EXTENSIONS=$($CODE_SERVER --list-extensions $EXTENSION_ARG)
+readarray -t EXTENSIONS_ARRAY <<< "$LIST_EXTENSIONS"
+function extension_installed() {
+  if [ "${USE_CACHED_EXTENSIONS}" != true ]; then
+    return 1
+  fi
+  for _extension in "$${EXTENSIONS_ARRAY[@]}"; do
```

Comment:
Should this be:

```suggestion
  for _extension in "${EXTENSIONS_ARRAY[@]}"; do
```

Reply:
I have to escape the $ as this is a templated script.

Reply:
Ohhhhhhhhhhhhhhhh right that makes sense, duh!